B/Bobby/Deps.toml

```diff
@@ -0,0 +1,5 @@
+[0]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
+Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Can probably be a test-only dependency.

Reply:
yes, thanks for pointing out! will change for next release